lua/weapons/weapon_ttt_rifle_stn.lua

```diff
@@ -0,0 +1,230 @@
+-- Traq Rifle created by https://steamcommunity.com/profiles/76561198813867340
+
+include("autorun/nl_library.lua")
+
+if SERVER then
+    AddCSLuaFile()
+end
+
+DEFINE_BASECLASS("weapon_tttbase")
```

Comment:
Same as what I wrote about the M16 - if you want this to work exactly like the Rifle but with a few tweaks, you can use the Rifle SWEP as the baseclass. Then you don't need to copy/rewrite everything from the Rifle, you can just overwrite what you need to.